### PR TITLE
Added Power BI Embedded client

### PR DESCRIPTION
**Pull request by wbreza**
Adding Power BI Embedded client

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@wbreza__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (wabreza). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by wbreza** [comment]
What does "Waiting for status to be reported" mean?  Anything I can do about this?

**Comment by japj** [comment]
I think travis might have had some backlog, it seems the build has just recently started?

**Comment by wbreza** [comment]
@japj - I recently pushed a new commit to resolve conflicts on my branch.  Build is passed now but i still see "Waiting for status to be reported"...

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by wbreza** [comment]
Ping

**Comment by markcowl** [comment]
@wbreza This branch is open again, please update, and miic the project.json changes in the other netcore clients

**Comment by wbreza** [comment]
@markcowl - All rebased and ready to go.

File: src/ResourceManagement/PowerBIEmbedded/Microsoft.Azure.Management.PowerBIEmbedded/project.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "keyFile": "../../../../tools/MSSharedLibKey.snk"
+    },
+
+    "dependencies": {
+        "Microsoft.Rest.ClientRuntime.Azure": "[4.0.2,5.0.0)"
+    },
+
+    "frameworks": {
+        "net45": {
+        },
```
Comment (markcowl) [inline src/ResourceManagement/PowerBIEmbedded/Microsoft.Azure.Management.PowerBIEmbedded/project.json]: See the Compute project as an example.  We are using ClientRuntime.Azure >= 3.1.0 for net 45 and >= 3.3.0 for the netstandard targets

File: src/ResourceManagement/PowerBIEmbedded/PowerBIEmbedded.Tests/project.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                "type": "platform",
+                "version": "1.0.0-rc2-*"
+            }
+        }
+    }
+  },
+    "dependencies": {
+        "Microsoft.Azure.Test.HttpRecorder": "[1.6.1-preview,2.0.0)",
+        "Microsoft.Rest.ClientRuntime.Azure.TestFramework": "[1.2.2-preview,2.0.0)",
+        "Microsoft.Rest.ClientRuntime.Azure": "[4.0.2,5.0.0)",
```
Comment (markcowl) [inline src/ResourceManagement/PowerBIEmbedded/PowerBIEmbedded.Tests/project.json]: [3.3.0, 4.0.0]

File: src/ResourceManagement/Storage/Microsoft.Azure.Management.Storage/generate.cmd [hunk 3]
```diff
@@ -3,6 +3,8 @@
 :: Copyright (C) Microsoft Corporation. All Rights Reserved.
```
Comment (markcowl) [inline src/ResourceManagement/Storage/Microsoft.Azure.Management.Storage/generate.cmd]: Please remove this change

**Comment by markcowl** [comment]
@wbreza please make changes as specified in comments.

File: src/ResourceManagement/Storage/Microsoft.Azure.Management.Storage/generate.cmd (hunk 3 above)
Comment (wbreza) [inline src/ResourceManagement/Storage/Microsoft.Azure.Management.Storage/generate.cmd]: Removed

File: src/ResourceManagement/PowerBIEmbedded/PowerBIEmbedded.Tests/project.json (hunk 2 above)
Comment (wbreza) [inline src/ResourceManagement/PowerBIEmbedded/PowerBIEmbedded.Tests/project.json]: Updated

File: src/ResourceManagement/PowerBIEmbedded/Microsoft.Azure.Management.PowerBIEmbedded/project.json (hunk 1 above)
Comment (wbreza) [inline src/ResourceManagement/PowerBIEmbedded/Microsoft.Azure.Management.PowerBIEmbedded/project.json]: Updated

**Comment by wbreza** [comment]
@markcowl - Ready for review

File: src/ResourceManagement/PowerBIEmbedded/PowerBIEmbedded.Tests/ScenarioTests/PowerBIEmbeddedTests.cs [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿using Xunit;
+
+namespace PowerBIEmbedded.Tests.Tests
+{
+    public class PowerBIEmbeddedTests
```
Comment (markcowl) [inline src/ResourceManagement/PowerBIEmbedded/PowerBIEmbedded.Tests/ScenarioTests/PowerBIEmbeddedTests.cs]: No tests?

**Comment by markcowl** [comment]
@wbreza Don't know how I missed it before, but looks like there are no tests?  We need to have tests here before we can accxept a check-in to the AutoRest branch.

**Comment by wbreza** [comment]
Added unit tests

**Comment by wbreza** [comment]
@markcowl ping

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Pull request merged**